jujugui/static/gui/src/app/components/deployment-bar/deployment-bar.js

```diff
@@ -22,10 +22,16 @@ YUI.add('deployment-bar', function() {
 
   juju.components.DeploymentBar = React.createClass({
     propTypes: {
+      changeState: React.PropTypes.func.isRequired,
+      currentChangeSet: React.PropTypes.object.isRequired,
```

Comment:
As discussed, the value of that method is passed in and is updated with the calls here: https://github.com/juju/juju-gui/pull/1472/files#diff-21dd467cb411bdd922556d1af7461864R429

I've renamed the hasCommits method and variable to be more clear about its purpose.